src/modules/users/consumers/PermissionConsumer.ts

```diff
@@ -40,7 +44,9 @@ export class PermissionConsumer extends AbstractAmqpQueueConsumer<
       },
       {
         creationConfig: {
+          exchange: PermissionConsumer.EXCHANGE_NAME,
           queueName: PermissionConsumer.QUEUE_NAME,
+          topicPattern: '',
```

Comment:
@CarlosGamero Should we make this parameter optional and default to ''? It is mandatory for queue binding operation in RabbitMQ, but empty value is acceptable.

Reply:
I am not sure what are the implications of having it empty, but if it is a valid value the can be use in "real world" cases I would vote for having it optional to simplify the API

Reply:
empty is treated same as any other value, so you will get messages if both consumer and publisher are configured to use empty topic. this means that routing will be done on a purely exchange/queue level.

Reply:
I see, thanks for explaining! 🙏 So I agree with having it optional 🙌